test/performance/odp_l2fwd.c

```diff
@@ -95,6 +95,9 @@ typedef struct {
 	/* Some extra features (e.g. error checks) have been enabled */
 	uint8_t extra_feat;
 
+	/* Has some state that needs to be maintained across tx and/or rx */
+	uint8_t has_state;
```

Comment:
Not sure if the `state` abstraction is necessary. Referencing to TX completion directly would be clearer when reading the code.

Reply:
The `state` should provide some future-proofness in order to avoid a similar kind of mess that cumulated before `process_extra_features()` existed.

Reply:
OK, would have preferred to have everything inside `process_extra_features()`, but indeed prefetching makes that difficult.